### DISCUSSION TITLE
[7주차] [온라인 스터디] 글 모음

### DISCUSSION THREAD

**Discussion by bigsaigon333** [fe-backpackers/fe-interview-basecamp#221] [General]
[7주차] [온라인 스터디] 글 모음

**Comment by bigsaigon333** [comment]
현재 기분을 10점 만점으로 표현해주세요 :smile: + 이유도 기재해주세요 :smile:

**Reply by bigsaigon333** [reply]
7점 이번주 면접 3개가 있어서, 매우 부담스럽고 바쁜 주간이네요 :dizzy_face:  빨리 일하고 싶습니다!

**Reply by MINGDY98** [reply]
6점 끊임없이 공부해야한다는 생각에 머리 아파요 ㅠ0ㅠ
